src/web/rustdoc.rs

```diff
@@ -271,26 +295,18 @@ impl RustdocPage {
 
         let mut response = Response::with((Status::Ok, html));
         response.headers.set(ContentType::html());
-        if is_latest_url {
-            response
-                .headers
-                .set(CacheControl(vec![CacheDirective::MaxAge(0)]));
+        if let Some(cache_control) = if is_latest_url {
```

Comment:
This `if let ... = if ... {` construction is pretty confusing to me. I think it would be clearer to split it out into an assignment:

```
let cache_control = if is_latest_url {
  ...
} else {
  ...
};

if let Some(cc) = cache_control {
  response.headers.set(cache_control}
}
```

Reply:
I updated it